src/graphql/operations/vp.ts

```diff
@@ -4,6 +4,10 @@ import db from '../../helpers/mysql';
 const scoreAPIUrl = process.env.SCORE_API_URL || 'https://score.snapshot.org';
 
 export default async function (_parent, { voter, space, proposal }) {
+  if (voter === '0x0000000000000000000000000000000000000000' || voter === '') {
+    return Promise.reject(new Error('invalid address'));
+  }
```

Comment:
Maybe also better to check `getAddress` to check checksum

Reply:
We could, but this will require adding a `address` dependency